servatrice/src/servatrice_database_interface.cpp

```diff
@@ -831,31 +832,35 @@ void Servatrice_DatabaseInterface::logMessage(const int senderId, const QString
     execSqlQuery(query);
 }
 
-bool Servatrice_DatabaseInterface::changeUserPassword(const QString &user, const QString &oldPassword, const QString &newPassword)
+bool Servatrice_DatabaseInterface::changeUserPassword(const QString &user, const QString &oldPassword, const QString &newPassword, bool force)
```

Comment:
The original behavior of the function returned false if the results were successful.  Seemed back words to me.  The caller was sending a function call to "changeUserPassword" and it would return "false" if the check actually succeeded as the original variable name was "failedtochangepassword".  If some one else called the function for other uses with just "changeuserpassword" and expected a true for success or false for fail they wouldnt receive those results which was confusing.